CMakeLists.txt

```diff
@@ -2,19 +2,19 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
 
 SET(FC "/usr/bin/gfortran" CACHE STRING "Fortran Compiler")
 SET(FFLAGS "-cpp" CACHE STRING "Fortran compiler flags")
```

Comment:
As in the FTOL PR, you could consider adding the same flags as they are used in the hand-crafted Makefile for consistency.

Reply:
The FFLAGS have been set to `-cpp -O`, consistent with the HOHQMesh Makefile.

Reply:
Thanks!